sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/SonarAnalysisContextTest.cs

```diff
@@ -54,10 +55,13 @@ public void SonarAnalysis_WhenShouldAnalysisBeDisabledReturnsTrue_NoIssueReporte
 
             try
             {
-                foreach (var testCase in this.TestCases)
+                foreach (var testCase in TestCases)
                 {
                     // ToDo: We should find a way to ack the fact the action was not run
-                    Verifier.VerifyNoIssueReported(testCase.Path, testCase.Analyzer, ParseOptionsHelper.FromCSharp8);
+                    Verifier.VerifyNoIssueReported(testCase.Path,
+                                                   testCase.Analyzer,
+                                                   ParseOptionsHelper.FromCSharp8,
+                                                   additionalReferences: MetadataReferenceFacade.GetSystemComponentModelPrimitives());
```

Comment:
is this metadata necessary for a specific framework or for all?
maybe add a comment?

Reply:
I think comments will make the code harder to maintain since we have to maintain both the code and the comments. Also, the answer can be easily found by checking the implementation.